io/pci/pci_info_lscfg.py.data/Readme.txt

```diff
@@ -0,0 +1,3 @@
+This Test fetch the information from lscfg of pci devices using 
```

Comment:
This test extracts PCI device related information (slot, subvendor and device ID) using lscfg command and compares it with information obtained using sysfs entries. In case of a mis-match a failure will be flagged.
This test is repeated for all PCI devices available on the server.